pkg/dataload/value.go

```diff
@@ -21,6 +21,12 @@ type DataLoadInfo struct {
 
 	// Image specifies the image that the DataLoad job uses
 	Image string `yaml:"image,omitempty"`
+
+	// LoadMemoryData specifies if the dataload job should load memory or not
+	LoadMemoryData bool `yaml:"loadMemoryData,omitempty"`
```

Comment:
Do we need define this in the struct  for the comon dataloadInfo Object?

Reply:
thx,  so define JindoOptions to include these two parameter, not in the common dataloadInfo